nsqadmin/README.md

```diff
@@ -12,6 +12,6 @@ Read the [docs](http://nsq.io/components/nsqadmin.html)
  3. `$ gulp clean watch` or `$ gulp clean build`
  4. `$ go-bindata --debug --pkg=nsqadmin --prefix=static/build static/build/...`
  5. `$ cd ../nsqadmin && go build && ./nsqadmin`
- 6. make changes
+ 6. make changes (and repeat steps 4 and 5 to see changes)
```

Comment:
This isn't _always_ true (specifically, it's only true when you're making changes to the Go code of `nsqadmin`).  Another way of communicating this is to say `(repeat step 5 if you make changes to any Go code)`